test/functional/lua/fs_spec.lua

```diff
@@ -285,12 +285,18 @@ describe('vim.fs', function()
     it('works with backward slashes', function()
       eq('C:/Users/jdoe', vim.fs.normalize('C:\\Users\\jdoe'))
     end)
+    it('works with mixed forward and backward slashes', function()
+      eq('C:/Users/jdoe', vim.fs.normalize('C:\\//Users//\\jdoe//\\'))
```

Comment:
The single-letter string represented as `\\` instead of `\` due to escaping is quite confusing given that we now want to deal with double slashes. Can you change `'...\\...'` with `[[...\...]]` (including existing test cases)? This will make tests more readable.

Reply:
Fair enough. I'll change all of them into raw string notation.

Reply:
@dundargoc